cluster/ci/monitoring/mixins/Makefile

```diff
@@ -26,6 +27,10 @@ tide.json:
 	@echo "Generating tide.json ..."
 	jsonnet -J vendor/ grafana_dashboards/tide.jsonnet > ../$@
 
+prometheus_rule_prow.yaml:
+	@echo "Generating prometheus_rule_prow.yaml ..."
+	jsonnet  ./prometheus/prometheus_rule_prow.jsonnet | gojsontoyaml > ../$@
```

Comment:
Do we need to install this in the container image for the presubmit?

Reply:
You can see that our tide did not complain.
[This tool is added already](https://github.com/openshift/release/blob/master/projects/origin-release/dashboards-validation/Dockerfile#L9) by our friend @paulfantom from monitoring team. ^_^

Reply:
sweet!